### ISSUE TITLE
REQUEST: Ability to monitor notify push

### ISSUE THREAD

**Issue by seanmcquade20** [Alkaar/resy-booking-bot#55]
Hi Alkaar,

This bot is great. I've been able to use it to get reservations when I know in advance exactly what times the reservations will open. A request from me would be the ability to monitor restaurants where you set a "Notify". Sometimes I'll get a push from Resy but by the time I see it, open the app and try to reserve, it's already gone, so would be good if the bot could get these pushes and "wake up" to book.

Thanks
Sean

**Comment by Alkaar** [comment]
This is a reasonable feature request. Though I don't know if I want to take this on at this moment. It would require the bot polling an e-mail account or being able to receive a push notification or text message. On top of that, I'm not sure what form the message would look like. Reservations I can easily iterate on but notifies have a longer testing cycle.

Currently, there are also other projects and things I want to take on. However, I will leave this issue open in case my mind changes on this.

**Comment by seanmcquade20** [comment]
Sounds good - I appreciate you even considering it. Thanks for the work you're doing to provide this to us.

**Comment by justinishikawa** [comment]
> This is a reasonable feature request. Though I don't know if I want to take this on at this moment. It would require the bot polling an e-mail account or being able to receive a push notification or text message. On top of that, I'm not sure what form the message would look like. Reservations I can easily iterate on but notifies have a longer testing cycle.

Have you seen any spec or code for how this push message comes in?

**Comment by Alkaar** [comment]
Nope, and I highly doubt I'd find any documentation on it. Even this bot I didn't really look up any documentation. I just inspected what data was coming and going and reverse-engineered it from there. I only found the documentation after the fact and I link it in the code just as a reference.
